_ml-commons-plugin/extensibility/blueprints.md

```diff
@@ -73,7 +73,97 @@ The `action` parameter supports the following options.
+| `post_process_function` | String | Optional. A built-in or custom Painless script to post-process the model output data. OpenSearch provides the following built-in post-process functions that you can call directly:<br> - `connector.pre_process.cohere.embedding` for [Cohere text embedding models](https://docs.cohere.com/reference/embed)<br> - `connector.pre_process.openai.embedding` for [OpenAI text embedding models](https://platform.openai.com/docs/api-reference/embeddings) <br> - `connector.post_process.default.embedding` that you can use to post-process documents in the model response so that they are in the format that neural search expects (OpenSearch 2.11 or later). For more information, see [built-in functions](#built-in-pre--and-post-processing-functions).  |
+
+## Built-in pre- and post-processing functions
+
+Call the built-in pre- and post-processing functions instead of writing a custom Painless script when connecting to the following text embedding models or your own text embedding models deployed on a remote server (for example, Amazon SageMaker):
+
+- [Pretrained models provided by OpenSearch](https://opensearch.org/docs/latest/ml-commons-plugin/pretrained-models/)
+- [OpenAI remote models](https://platform.openai.com/docs/api-reference/embeddings)
+- [Cohere remote models](https://docs.cohere.com/reference/embed)
+
+OpenSearch provides the following pre- and post-processing functions:
+
+- OpenAI: `connector.pre_process.openai.embedding` and `connector.post_process.openai.embedding`
+- Cohere: `connector.pre_process.cohere.embedding` and `connector.post_process.cohere.embedding`
+- [Default](#default-pre--and-post-processing-functions) (for neural search): `connector.pre_process.default.embedding` and `connector.post_process.default.embedding`
+
+### Default pre- and post-processing functions
+
+The default pre- and post-processing functions translate between the format that the model expects and the format that [neural search]({{site.url}}{{site.baseurl}}/search-plugins/neural-search/) expects. 
+
+#### Example request
+
+The following example request creates a SageMaker text embedding connector and calls the default post-processing function:
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "Sagemaker text embedding connector",
+  "description": "The connector to Sagemaker",
+  "version": 1,
+  "protocol": "aws_sigv4",
+  "credential": {
+    "access_key": "<REPLACE WITH SAGEMAKER ACCESS KEY>",
+    "secret_key": "<REPLACE WITH SAGEMAKER SECRET KEY>"
```

Comment:
Same for another comment https://github.com/opensearch-project/documentation-website/pull/5168/files#r1356002365, please add "session_token" field